--- poppler-0.22.0/glib/Makefile.am.linkage	2012-12-15 16:38:53.000000000 +0100
+++ poppler-0.22.0/glib/Makefile.am	2013-01-11 21:59:20.329022470 +0100
@@ -97,7 +97,7 @@ introspection_files = $(filter-out poppl
 Poppler-0.18.gir: libpoppler-glib.la
 Poppler_0_18_gir_INCLUDES = GObject-2.0 Gio-2.0 cairo-1.0
 Poppler_0_18_gir_CFLAGS = $(INCLUDES) -I$(top_builddir)
-Poppler_0_18_gir_LIBS = libpoppler-glib.la
+Poppler_0_18_gir_LIBS = libpoppler-glib.la gmodule-2.0
 Poppler_0_18_gir_FILES = $(introspection_files)
 Poppler_0_18_gir_SCANNERFLAGS = --c-include poppler.h
 Poppler_0_18_gir_EXPORT_PACKAGES = poppler-glib
