diff -up poppler-21.04.0/CMakeLists.txt.omv~ poppler-21.04.0/CMakeLists.txt
--- poppler-21.04.0/CMakeLists.txt.omv~	2021-04-08 00:03:08.077082108 +0200
+++ poppler-21.04.0/CMakeLists.txt	2021-04-08 00:03:12.140383100 +0200
@@ -1,6 +1,6 @@
 project(poppler)
 
-cmake_minimum_required(VERSION 3.10.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
 
